src/si/ratio.rs

```diff
@@ -32,49 +32,117 @@ quantity! {
     }
 }
 
-/// Implementation of various stdlib inverse trigonometric functions
+/// Implementation of various stdlib functions
```

Comment:
One minor change and I'll merge!

```suggestion
/// Implementation of various stdlib functions.
```

Reply:
Done, via rebase, fixing the commit that changed that line in the first place.

Reply:
Thanks! I kicked of the test workflows again and will merge the PR once they're complete.